src/sentry/static/sentry/app/views/settings/components/forms/selectField.jsx

```diff
@@ -60,23 +60,13 @@ export default class SelectField extends React.Component {
       <InputField
         {...otherProps}
         alignRight={this.props.small}
-        field={({onChange, onBlur, disabled, required: _required, ...props}) => {
-          // We remove the required property here since applying it to the
-          // DOM causes the native tooltip to render in strange places.
-          const choices = getChoices(props);
-
+        field={({onChange, onBlur, required: _required, ...props}) => {
```

Comment:
I'd create a function to filter the props, instead of creating `_*`

Reply:
I prefer being able to pick and omit props in a single line